tfhe/Cargo.toml

```diff
@@ -17,7 +17,7 @@ exclude = [
     "/js_on_wasm_tests/",
     "/web_wasm_parallel_tests/",
 ]
-rust-version = "1.73"
+rust-version = "1.75"
```

Comment:
we will likely take the most recent rust when we agree we go for v1, this is good for now 👍